tests/unit/test-wcs-notifications-debug-tool.php

```diff
@@ -14,17 +39,8 @@ class WCS_Subscription_Notifications_Debug_Tool_Test extends WP_UnitTestCase {
 	 */
 	public function test_process_batch_notifications() {
 
-		// Enable feature.
-		update_option( WC_Subscriptions_Admin::$option_prefix . WC_Subscriptions_Email_Notifications::$switch_setting_string, 'yes' );
-		update_option(
-			WC_Subscriptions_Admin::$option_prefix . WC_Subscriptions_Email_Notifications::$offset_setting_string,
-			[
-				'number' => '3',
-				'unit'   => 'days',
-			]
-		);
-
-		$batches = $this->notification_subscription_provider();
+		$batches   = $this->notification_subscription_data_provider();
+		$processor = new WCS_Notifications_Debug_Tool_Processor();
```

Comment:
Might be better to call `\WCS_Action_Scheduler_Customer_Notifications::get_action_args` in the `$action_args` below.

Reply:
Done https://github.com/Automattic/woocommerce-subscriptions-core/pull/689/commits/f9bee572a8cdd2f32271bfa2a4a05b743ca7ba54